app/forms/developers_search_form.rb

```diff
@@ -1,19 +1,16 @@
 class DevelopersSearchForm
   include Virtus.model
   include ActiveModel::Model
-  include ActiveRecord::Sanitization::ClassMethods
 
   attribute :language, String
   attribute :programming_language, String
 
   def search
-    check_language = language.present?
-    check_programming_language = programming_language.present?
-    return Developer.none if !check_language && !check_programming_language
+    return Developer.none if [language, programming_language].all?(&:blank?)
 
     query = Developer.includes(:developer_languages, :languages, :developer_programming_languages, :programming_languages)
```

Comment:
Do you need all of the includes?

Reply:
Thanks you very much!
Sorry i missed this case, i have removed the unnecessary includes.